e2e/fixtures/auth.ts

```
@@ -0,0 +1,83 @@
+      password: process.env.TEST_USER_PASSWORD!,
+      user_metadata: { is_test_user: true }
+    });
+
+    if (error) throw error;
+    if (!data.user) throw new Error('Failed to create test user: No user data returned');
+
+    const user = { id: data.user.id, email: data.user.email! };
+    testUsers.set(email, user);
+    return user;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)